### PR TITLE
WEBDOM-391: Implement https://github.com/digipolisgent/symfony_bundle_domainator9k-core/pull/8.

### PR DESCRIPTION
**Pull request by Jelle-S**
Implement https://github.com/digipolisgent/symfony_bundle_domainator9k-core/pull/8.

### CODE REVIEW

File: Provisioner/BuildProvisioner.php [hunk 1]
```diff
@@ -271,14 +265,18 @@ protected function createCrontab(SSH2 $ssh, ApplicationEnvironment $applicationE
...
 
-            $this->taskService->addSuccessLogMessage($this->task, 'Crontab created.', 2);
+            $this->taskLoggerService->addSuccessLogMessage($this->task, 'Crontab created.', 2);
         } catch (\Exception $ex) {
-            $this->taskService
+            $this->taskLoggerService
                 ->addErrorLogMessage($this->task, $ex->getMessage(), 2)
                 ->addFailedLogMessage($this->task, 'Creating crontab failed.', 2);
 
             throw $ex;
```
Comment (matthijs-va) [inline Provisioner/BuildProvisioner.php:274]: LoggedException

File: Provisioner/BuildProvisioner.php [hunk 2]
```diff
@@ -204,10 +198,10 @@ protected function createFiles(SSH2 $ssh, ApplicationEnvironment $applicationEnv
...
                 $this->executeSshCommand($ssh, $command);
 
-                $this->taskService->addSuccessLogMessage($this->task, 'Files created.', 2);
+                $this->taskLoggerService->addSuccessLogMessage($this->task, 'Files created.', 2);
             }
         } catch (\Exception $ex) {
-            $this->taskService
+            $this->taskLoggerService
                 ->addErrorLogMessage($this->task, $ex->getMessage(), 2)
                 ->addFailedLogMessage($this->task, 'Creating files failed.', 2);
```
Comment (matthijs-va) [inline Provisioner/BuildProvisioner.php:207]: throw logged exception

File: Provisioner/BuildProvisioner.php [hunk 3]
```diff
@@ -150,9 +144,9 @@ protected function createSymlinks(SSH2 $ssh, ApplicationEnvironment $application
                 $this->executeSshCommand($ssh, $command);
             }
 
-            $this->taskService->addSuccessLogMessage($this->task, 'Symlinks created.', 2);
+            $this->taskLoggerService->addSuccessLogMessage($this->task, 'Symlinks created.', 2);
         } catch (\Exception $ex) {
-            $this->taskService
+            $this->taskLoggerService
                 ->addErrorLogMessage($this->task, $ex->getMessage(), 2)
                 ->addFailedLogMessage($this->task, 'Creating symlinks failed.', 2);
```
Comment (matthijs-va) [inline Provisioner/BuildProvisioner.php:152]: throw logged exception

File: Provisioner/BuildProvisioner.php [hunk 4]
```diff
@@ -98,9 +92,9 @@ protected function createFolders(SSH2 $ssh, ApplicationEnvironment $applicationE
                 $this->executeSshCommand($ssh, 'mkdir -p -m 750 ' . escapeshellarg($path));
             }
 
-            $this->taskService->addSuccessLogMessage($this->task, 'Directories created.', 2);
+            $this->taskLoggerService->addSuccessLogMessage($this->task, 'Directories created.', 2);
```
Comment (matthijs-va) [inline Provisioner/BuildProvisioner.php:95]: throw logged exception

**Review by matthijs-va — CHANGES_REQUESTED** [review]

File: Provisioner/BuildProvisioner.php (hunk 1 above)
Comment (Jelle-S) [reply Provisioner/BuildProvisioner.php:274]: This exception is caught in the `doRun` method and re-thrown as a LoggedException there.

File: Provisioner/BuildProvisioner.php (hunk 2 above)
Comment (Jelle-S) [reply Provisioner/BuildProvisioner.php:207]: This exception is caught in the `doRun` method and re-thrown as a LoggedException there.

File: Provisioner/BuildProvisioner.php (hunk 3 above)
Comment (Jelle-S) [reply Provisioner/BuildProvisioner.php:152]: This exception is caught in the `doRun` method and re-thrown as a LoggedException there.

File: Provisioner/BuildProvisioner.php (hunk 4 above)
Comment (Jelle-S) [reply Provisioner/BuildProvisioner.php:95]: This exception is caught in the `doRun` method and re-thrown as a LoggedException there.

**Review by matthijs-va — APPROVED** [review]

**Pull request merged**